node_modules/gh-orgunit/lib/api.js

```diff
@@ -577,6 +580,145 @@ var deleteOrgUnitSeries = module.exports.deleteOrgUnitSeries = function(ctx, id,
 };
 
 
+//////////////
+// Calendar //
+//////////////
+
+/**
+ * Get the calendar for an organisational unit
+ *
+ * @param  {Context}        ctx                 Standard context containing the current user and the current app
+ * @param  {Number}         id                  The id of the organisational unit to get the calendar for
+ * @param  {String}         start               The timestamp (ISO 8601) from which to get the calendar for the organisation unit
+ * @param  {String}         end                 The timestamp (ISO 8601) until which to get the calendar for the organisation unit
+ * @param  {Function}       callback            Standard callback function
+ * @param  {Object}         callback.err        An error object, if any
+ * @param  {Event[]}        callback.events     The requested organisational unit calendar
+ */
+var getOrgUnitCalendar = module.exports.getOrgUnitCalendar = function(ctx, id, start, end, callback) {
+    // Ensure that the organisational unit id is a valid number
+    id = GrasshopperUtil.getNumberParam(id);
+
+    var validator = new Validator();
+    validator.check(id, {'code': 400, 'msg': 'A valid organisational unit id must be provided'}).isInt();
+    validator.check(start, {'code': 400, 'msg': 'A start time must be provided in a valid date format'}).isDate();
```

Comment:
Although we only claim to support ISO 8601, it can be anything that `Date.parse` can handle. As that's what sequelize and validator.js use under the hood